src/utils/arrow_hacks/vector_selection.cc

```diff
@@ -30,8 +30,8 @@
 // @nicholas: these headers are hidden from Arrow API
 #include <iostream>
 
-#include "/Users/corrado/hustle/arrow/cpp/src/arrow/compute/kernels/common.h"
-#include "/Users/corrado/hustle/arrow/cpp/src/arrow/compute/kernels/util_internal.h"
+#include "../../arrow/cpp/src/arrow/compute/kernels/common.h"
```

Comment:
Minor Nit: You can write it as "arrow/cpp/src/<rest of the path>" instead of ../../ in the beginning.

Reply:
I see - It's interesting that the VSCode does not recognize `"arrow/cpp/src/"` prefix. Shall I tweak the VSCode setting to include `arrow/`as one of the path?

Reply:
Oh okay, I think these files were changed for experiments in local machine based on the conversation I had with Nicholas. However, we are not using this module (hustle_src_utils_arrow_hacks) anymore in the build. That's why I think this path was not updated. 


I think we can do two things,

* We can archive this module somehow so that this can be referenced in the future if needed.
* Or we may need to make this module compatible with the arrow version that we use by using the functions that are compatible with the current arrow function. (I could see only one helper function has a problem).

We can solve this in another PR if needed!